game/models.py

```
@@ -581,4 +583,30 @@ def __str__(self):
+    Vercel's ephemeral serverless filesystem without requiring external
+    object storage or a persistent MEDIA_ROOT directory.
+    """
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="profile"
+    )
+    # Stored as a base64 data URI. Empty string means no avatar set.
+    avatar = models.TextField(blank=True, default="")
+
+    def __str__(self):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)